package.json

```diff
@@ -46,11 +48,13 @@
     "gruntfile-editor": "^0.2.0",
     "inquirer": "^0.8.0",
     "isbinaryfile": "^2.0.0",
+    "istextorbinary": "^1.0.2",
```

Comment:
Problem is that `isbinaryfile` needs a file as input, and we only have a buffer in `conflicter.js` ... This could be refactor to have a file, but i'm not sure it make sense ...